src/loaders/storage/loadRlLoadout.ts

```diff
@@ -51,6 +51,8 @@ export async function loadRlLoadout(manager: RocketAssetManager, player: Extende
       phongMaterial.color = oldMaterial.color
       phongMaterial.shininess = (1 - oldMaterial.roughness) * 100
       phongMaterial.skinning = oldMaterial.skinning
+      phongMaterial.reflectivity = 0
```

Comment:
shouldn't reflectivity depend on roughness?

Reply:
Do you have a suggestion? Feel free to test it out and let me know what you think fits the best. After testing different values it seemed to me that reflectivity wasn't affecting the "roughness" of Phong material as much as shininess was.

Reply:
I'll test it later

Reply:
I'm not sure how the PhongMaterial should behave but I think it is currently completely ignoring the envmap when it comes to reflections. When switched to StandardMaterial the reflections look completely different.

Reply:
This might be related:
https://github.com/mrdoob/three.js/issues/9550#issuecomment-241162868